lib/install/management/store_files.go

```diff
@@ -126,7 +126,7 @@ func (d *Dispatcher) deleteDatastoreFiles(ds *object.Datastore, path string, for
 	}
 
 	m := object.NewFileManager(ds.Client())
-	if d.isVSAN(ds) {
+	if d.geVC65() || d.isVSAN(ds) {
```

Comment:
Here, the top level method to delete should be method 
```func (d *Dispatcher) deleteDatastoreFiles(ds *object.Datastore, path string, force bool) (bool, error)```
So caller doesn't need to care about the internal impl, I think.

The last comment is same to @fdawg4l mentioned above. But I think delete interactively is not an optimization. It will slow down the delete.
But if vsphere is going this way, and we don't care too much about 6.0, it's also fine to remove this branch.